tests/testthat/test-calculate_stay.R

```diff
@@ -235,22 +235,28 @@ test_that("Can calculate the correct stay for vectors of dates", {
   )
 })
 
-test_that("normal calculate stay function works", {
-  test_tibble_norm <- tibble::tribble(
-    ~start_date, ~end_date,
-    2019 - 03 - 31, 2019 - 10 - 31,
-    2019 - 06 - 30, 2019 - 08 - 31,
-    2019 - 01 - 01, 2020 - 04 - 01,
-    2019 - 04 - 01, 2020 - 07 - 01,
-    2019 - 03 - 31, NA,
-    2019 - 06 - 30, NA,
-    2019 - 01 - 01, NA,
-    2019 - 04 - 01, NA
-  )
```

Comment:
@Jennit07 The issue was with the tribble definition (so my fault!) The dates weren't being declared properly.